transfer_transaction_test.go

```diff
@@ -6,7 +6,7 @@ import (
 	"testing"
 )
 
-func TestCryptoTransferTransaction_Execute(t *testing.T) {
+func TestIntegrationCryptoTransferTransaction(t *testing.T) {
```

Comment:
incorrect test name; in this particular case we shouldn't be using `CryptoTransferTransaction` as the name since we don't use a `CryptoTransferTransaction` struct. Instead this should use `TransferTransaction`, so the name would look something like `TestIntegrationTransferTransactionCanTransferHbars`